### DISCUSSION TITLE
Release cycle update: Official security support extended by a year

### DISCUSSION THREAD

**Discussion by BHSPitMonkey** [oerdnj/deb.sury.org#2148] [General]
[PHP RFC: Release cycle update](https://wiki.php.net/rfc/release_cycle_update) appears to have passed a few weeks ago, and the original Supported Versions page was updated to reflect that security support for versions 8.1 and higher has been extended by approximately a year: 

https://www.php.net/supported-versions.php

Apologies if I missed an acknowledgement somewhere already, but is the PPA planned to match the new lifecycle? Are there unresolved questions that must be worked out before that can be confirmed?

**Comment by oerdnj** [comment]
I am confused - what is your question? This repository provides security updates for PHP versions as far as PHP 5.6. So, yes, this makes the job a little bit easier, but otherwise it doesn’t really affect anything.

**Reply by BHSPitMonkey** [reply]
Sorry for that! I wasn't sure what state those EOL packages are in, and if they're planned to stick around given the description on the PPA:

> Only Supported Versions of PHP (http://php.net/supported-versions.php) for Supported Ubuntu Releases (https://wiki.ubuntu.com/Releases) are provided. Don't ask for end-of-life PHP versions or Ubuntu release, they won't be provided.

If the aim is to continue meeting or exceeding the official  security support lifespans, that's great. Thanks for clarifying and for providing these builds over the years!

**Reply by oerdnj** [reply]
Oh, sorry, that’s some old note. The other part holds, the first one doesn’t.

**Reply by oerdnj** [reply]
I cleaned up the description now.
